plugins/com.contrastsecurity.ide.eclipse.core.tests/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,7 +1,7 @@
 eclipse.preferences.version=1
 org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
-org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.7
-org.eclipse.jdt.core.compiler.compliance=1.7
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.8
+org.eclipse.jdt.core.compiler.compliance=1.8
```

Comment:
This might vary whenever we compile or setup our projects. But remember we must try to make it compatible with Java 1.7 since that's one requirement.